sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -851,6 +874,7 @@ public async Task StartRecognizeReceiptsWithSupportedLocale(string locale)
         }
 
         [Test]
+        [Ignore("Not supported")]
         public void StartRecognizeReceiptsWithWrongLocale()
```

Comment:
Why do we have so many `Not supported`? Are we not supporting receipts and business cards anymore?

Reply:
The service hasn't deployed receipts and BC in their 2.1-preview.2 endpoints so we decided to work on a feature branch and disable those tests.
I will add this to the description of the PR.

Reply:
I see. I hadn't noticed this was a feature branch.